app/utils.py

```diff
@@ -0,0 +1,11 @@
+from flask import jsonify
+
+
+def error_missing_json_key(*args):
+    if len(args) == 1:
+        return jsonify({"msg": "Missing key `{}`".format(args[0]), "status": "error"}), 400
+    else:
+        return jsonify({"msg": "Missing a JSON key of name " + ", ".join(map(lambda x: "`{}`".format(x),
```

Comment:
This I can't really understand :confused:

Reply:
It's a function which returns missing keys in the supplied JSON as an error message.

Reply:
I'd vote for making it more verbose (without the lambda function) just because it'd make it a bit more readable but that's totally bikeshedding :bike: 

@juped could you give us some input on this?

Reply:
Might want to assemble the list of keys on one line and the error string on the next, though it's not really impenetrable as is